_episodes_rmd/07-vector-shapefile-attributes-in-r.Rmd

```diff
@@ -283,7 +283,7 @@ line_widths <- c(1, 2, 3, 4)
 
 We can use those line widths when we plot the data.
 
-```{r harv-paths-map-wide}
+```{r harv-paths-map-wide fig.cap="Roads and trails in the area demonstrating how to use different line thickness and colors."}
```

Comment:
```suggestion
```{r harv-paths-map-wide, fig.cap="Roads and trails in the area demonstrating how to use different line thickness and colors."}
```